Sources/MapboxMaps/Foundation/SubviewInteractionOnlyView.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+/// This class is a wrapper view which forwards all touch events to it's subviews
+public class SubviewInteractionOnlyView: UIView {
```

Comment:
This is public so that AnnotationView can inherit from it. I'd rather have AnnotationView inherit directly from UIView, keep this class internal, and duplicate the hitTest logic into AnnotationView.

Exposing this class as public leaks too much about the specific implementation we're going with right now and will make it more difficult if we want to take a different path in the future.